### PR TITLE
Fix publish docs

### PR DESCRIPTION
**Pull request by Skarlso**
### Description

Fixes https://github.com/weaveworks/eksctl/issues/5045

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

**Review by aclevername — APPROVED** [review]
how did this ever work?

Also isn't `publish-docs` in the `Makefile.docs`, not the standard `Makefile`?

so confused

**Comment by aclevername** [comment]
ah, it never eve worked :joy:

**Comment by Skarlso** [comment]
>Also isn't publish-docs in the Makefile.docs, not the standard Makefile?

The Makefile includes all other Makefiles at the top. :)

File: .github/workflows/publish-docs.yaml [hunk 1]
```diff
@@ -10,6 +10,11 @@ jobs:
     if: ${{ !github.event.release.prerelease }}
     runs-on: ubuntu-latest
     steps:
+      - name: Checkout
+        uses: actions/checkout@v2      
+        with:
+          token: ${{ secrets.WEAVEWORKSBOT_TOKEN }}
```
Comment (cPu1) [inline .github/workflows/publish-docs.yaml:16]: This isn't required.

**Review by cPu1 — APPROVED** [review]

**Comment by aclevername** [comment]
can we get another copy of this merged into the 0.91.0-rc.0 release branch, so it gets applied when we promote that release?

File: .github/workflows/publish-docs.yaml (hunk 1 above)
Comment (Skarlso) [reply .github/workflows/publish-docs.yaml:16]: Which on? The token? Oh, that's probably right. :)

Comment (cPu1) [reply .github/workflows/publish-docs.yaml:16]: Yeah, the token :)

**Comment by Skarlso** [comment]
> can we get another copy of this merged into the 0.91.0-rc.0 release branch, so it gets applied when we promote that release?

Sure, once this is merged, I'll backport it into the release branch.

**Pull request merged**